sdk/core/Azure.Core/src/Pipeline/Internal/Multipart/BufferedReadStream.cs

```diff
@@ -288,7 +293,7 @@ public bool EnsureBuffered(int minCount)
         {
             if (minCount > _buffer.Length)
             {
-                throw new ArgumentOutOfRangeException(nameof(minCount), minCount, "The value must be smaller than the buffer size: " + _buffer.Length.ToString());
+                throw new ArgumentOutOfRangeException(nameof(minCount), minCount, "The value must be smaller than the buffer size: " + _buffer.Length.ToString(CultureInfo.InvariantCulture));
```

Comment:
Curious: why add this?  I thought we didn't like to use InvariantCulture, but I don't remember why I think that.

Reply:
Ok, cool, I don't remember why I think that, so I could be completely wrong.

Do we know why the analyzer complains if we don't do it?  It would be good to understand why we're making this change.

Reply:
https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1305